tailwind.config.js

```diff
@@ -6,6 +6,13 @@ module.exports = {
     './src/**/*.tsx',
   ],
   theme: {
+    screens: {
+      sm: '640px',
+      md: '768px',
+      lg: '1024px',
+      xl: '1280px',
+      ultra: '1900px',
```

Comment:
that's quite a jump from xl to ultra. 
how about we add one more and apply more consistent naming (in tailwind world) and make use of traditionally recognized resolutions?

suggestion:
```
xl: 1280
2xl: 1440
3xl: 1920
```